app/assets/stylesheets/image_x.scss

```diff
@@ -71,3 +77,23 @@ $image-thumb-slider-border-color: #000;
 .#{$namespace}-image-x-label {
   text-align: center;
 }
+
+.#{$namespace}-frame {
+  position:absolute;
+  display:block;
+  border: 1px solid #000000;
+}
+
+.#{$namespace}-canvas {
+  position: absolute;
+  display: block;
+  border: 1px solid #000000;
+}
+
+.#{$namespace}-label {
+  width:100%;
```

Comment:
Properties should be ordered bottom, height, position, text-align, width<br>Colon after property should be followed by one space